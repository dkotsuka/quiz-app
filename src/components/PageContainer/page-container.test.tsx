import React from 'react'
import { Text } from 'react-native'
import { PageContainer } from '.'
import { create } from 'react-test-renderer';

describe("LayoutCoPageContainerntainer", () => {
    
    it("matches snapshot with padding", () => {
        const component = create(<PageContainer padding={10}>
            <Text>First component</Text>
            <Text>Second component</Text>
        </PageContainer>)
        expect(component).toMatchSnapshot()
    })

    it("matches snapshot with vertical padding", () => {
        const component = create(<PageContainer vPadding={10}>
            <Text>First component</Text>
            <Text>Second component</Text>
        </PageContainer>)
        expect(component).toMatchSnapshot()
    })

    it("matches snapshot with horizontal padding", () => {
        const component = create(<PageContainer hPadding={10}>
            <Text>First component</Text>
            <Text>Second component</Text>
        </PageContainer>)
        expect(component).toMatchSnapshot()
    })
})