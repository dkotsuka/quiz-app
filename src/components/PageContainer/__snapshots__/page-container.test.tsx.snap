// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutCoPageContainerntainer matches snapshot with horizontal padding 1`] = `
<RCTScrollView
  hPadding={10}
  style={
    Array [
      Object {
        "paddingHorizontal": 10,
      },
    ]
  }
>
  <View>
    <Text>
      First component
    </Text>
    <Text>
      Second component
    </Text>
  </View>
</RCTScrollView>
`;

exports[`LayoutCoPageContainerntainer matches snapshot with padding 1`] = `
<RCTScrollView
  padding={10}
  style={
    Array [
      Object {
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 10,
      },
    ]
  }
>
  <View>
    <Text>
      First component
    </Text>
    <Text>
      Second component
    </Text>
  </View>
</RCTScrollView>
`;

exports[`LayoutCoPageContainerntainer matches snapshot with vertical padding 1`] = `
<RCTScrollView
  style={
    Array [
      Object {
        "paddingVertical": 10,
      },
    ]
  }
  vPadding={10}
>
  <View>
    <Text>
      First component
    </Text>
    <Text>
      Second component
    </Text>
  </View>
</RCTScrollView>
`;
